src/components/Episode/Episode.jsx

```diff
@@ -53,6 +44,17 @@ export default function Episode(props){
 					thumbnail={ thumbnail }
 					trailer={ trailer }
 				/>
+				<div className={ s.content }>
+					<h2 className={ s.episode }>
```

Comment:
while technically having the `<h2>` before the `<h1>` is fine, maybe we should just make `.content` `display: flex` and have

```
.episode {
    order: 1;
}
.title {
   order: 2;
}
.body {
   order: 3;
}

Reply:
Yah, that's nice. Forgot flex could do that